### ISSUE TITLE
How can I add the wifi module's IP ?

### ISSUE THREAD

**Issue by gustavoguimas** [garbled1/balboa_homeassistan#12]
not quite sure how to add the ip of the module, what should I write on the config file ?

**Comment by garbled1** [comment]
This integration is a config_flow, so you access it on the integrations page in home assistant under configuration.  You can hit the + and search for balboa.  When it asks for the hostname, you can supply the IP there.

**Comment by gustavoguimas** [comment]
Awesome thanks

**Comment by garbled1** [comment]
Right, you have to go to the integrations page of home assistant and then add the integration

**Comment by saschakeller** [comment]
Thank you for the HACS integration of the Balbo Spa. I am still struggling with the same problem. The Balboa Spa Client is installed in HACS and in my home assistant I go to the integrations page and want to add the integration. But this one is not listed, so therefore I could not enter any IP adress. 
Do you habe any hints?

**Comment by orenatan** [comment]
> Thank you for the HACS integration of the Balbo Spa. I am still struggling with the same problem. The Balboa Spa Client is installed in HACS and in my home assistant I go to the integrations page and want to add the integration. But this one is not listed, so therefore I could not enter any IP adress.
> Do you habe any hints?

Hi 
Did you manage fixing it?

**Comment by orenatan** [comment]
> This integration is a config_flow, so you access it on the integrations page in home assistant under configuration. You can hit the + and search for balboa. When it asks for the hostname, you can supply the IP there.

Hi
What do you mean by the integration is config_flow. What should we do with it?

**Comment by saschakeller** [comment]
> Hi
Did you manage fixing it?> 

Not now. I have no prompt asking me for an IP adress. I could see the integration within the HACS menu in home assistant but there is no possibilibty to enter the IP. Ihad a look to the files within the folders but found not hint where to enter the adress.

**Comment by saschakeller** [comment]
Hi, 

I just tried again and resinstalled the intergration. After a restart I lcoud also integrate the configuration within home assistant and by doing that there was a prompt for the hosts ip adresse. So everythink is now working.
What I recognized is that since today I could also see a pictogram next to the HACS menu entry on the left side of the browsers view. This was not true the alst days. Maybe HACS hat not integrated well and therefore I had no chance to install the Balbo Spa.

Anyhow everything works. Thanks for setting up this software.
